pkg/config/config.go

```diff
@@ -114,7 +114,8 @@ func LoadFile(configPath string, relativePath ...string) (Config, error) {
 // updateRelativePaths updates relative paths in the config structure based on the provided relative path.
 func updateRelativePaths(relativePath string, config *Config) {
 	updatePath := func(path *string) {
-		if *path != "" && !filepath.IsAbs(*path) {
+		unifiedPath := *path
+		if *path != "" && !filepath.IsAbs(*path) && unifiedPath[0] != '/' {
```

Comment:
Why do we need the latter condition?

Reply:
because on Windows filepath.IsAbs("/cn_wallet.json") is false

Reply:
Create a separate test for Windows.

Reply:
You may add Windows-specific test as a separate file and use build constraint for it (https://pkg.go.dev/go/build#hdr-Build_Constraints) or check the platform in runtime: `if runtime.GOOS == "windows"`.

Reply:
Not resolved.